packages/trigger/src/index.ts

```diff
@@ -90,10 +96,12 @@ export class TriggerCache<T> {
  * dirty(1)
  * // this won't cause an update:
  * dirty(2)
+ * // this will cause an update to all keys:
+ * dirtyAll()
  */
 export function createTriggerCache<T>(
```

Comment:
Suggestion: We should deprecate this function, it creates needless deoptimization creating a new array and rebinding methods. We should recommend using the class directly.

Reply:
I'm fine with that. It's a primitive for advanced use anyway. But in another PR.